src/components/ActiveFile.tsx

```diff
@@ -0,0 +1,29 @@
+import "./ActiveFile.scss";
+
+import React from "react";
+import Stack from "../components/Stack";
+import { ToolButton } from "../components/ToolButton";
+import { save, file } from "../components/icons";
+import { t } from "../i18n";
```

Comment:
```suggestion
import React from "react";
import Stack from "../components/Stack";
import { ToolButton } from "../components/ToolButton";
import { save, file } from "../components/icons";
import { t } from "../i18n";

import "./ActiveFile.scss";
```

For consistency, as we are habituated seeing the first line `import React...` :)